superset/assets/visualizations/country_map.js

```diff
@@ -0,0 +1,190 @@
+require('./country_map.css');
+
+const d3 = require('d3');
+
+
+function countryMapChart(slice, payload) {
+  // CONSTANTES
+  let path;
+  let g;
+  let mapLayer;
+  let bigText;
+  let resultText;
+  const color = d3.scale.category20();
+  const container = slice.container;
+  const data = payload.data;
+  const maxMetric = d3.max(data, function (value) {
+    return value.metric;
+  });
+  const minMetric = d3.min(data, function (value) {
+    return value.metric;
+  });
+  const colorsByMetrics = d3.scale.linear()
+    .domain([maxMetric,
+      (maxMetric - minMetric) / 1.2,
+      (maxMetric - minMetric) / 1.5,
+      (maxMetric - minMetric) / 2,
+      (maxMetric - minMetric) / 3,
+      minMetric])
+    .range(['#d7191c', '#e76818', '#f29e2e', '#f9d057', '#ffff8c', '#90eb9d']);
```

Comment:
What about leveraging this?:
https://github.com/airbnb/superset/blob/master/superset/assets/javascripts/explorev2/stores/controls.jsx#L122
along with
https://github.com/airbnb/superset/blob/366ecefbaaf4cc31234cc981ebab7eb420efe019/superset/assets/javascripts/modules/colors.js#L69